### ISSUE TITLE
[Bug]: Error when installing PyMilvus

### ISSUE THREAD

**Issue by YiyunNi** [milvus-io/milvus#18529]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.1.0
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2): PyMilvus 2.1.0
- OS(Ubuntu or CentOS): Ubuntu
- CPU/Memory: N/A
- GPU: N/A
- Others: Python version 3.8.10
```

### Current Behavior

When I follow the instructions on [this page](https://milvus.io/docs/v2.1.x/install-pymilvus.md) to verify the installation of PyMilvus 2.1.0 with the code `python3 -c "from pymilvus import Collection"`, I got the error code in the screenshot.
![截屏2022-08-04 11 41 44](https://user-images.githubusercontent.com/74396087/182758697-203d38c8-041e-4909-81df-03280cfad740.png)

### Expected Behavior

According to the doc, "If PyMilvus is correctly installed, no exception will be raised when you run 'python3 -c "from pymilvus import Collection' "

### Steps To Reproduce

```markdown
1. Install PyMilvus with `$ python3 -m pip install pymilvus==2.1.0`.
2. Verify the installation with `$ python3 -c "from pymilvus import Collection"`
3. Got the error.
```

**Comment by jennyli-z** [comment]
python version? 3.?

**Comment by yanliang567** [comment]
@binbinlv is it a known issue?

/assign @binbinlv

**Comment by binbinlv** [comment]
Actually, I could not reproduce this issue at my side:
```
(milvus-binbin-1) root@milvus-client-pod:~/virtualenv# python3 -m pip install pymilvus==2.1.0
(milvus-binbin-1) root@milvus-client-pod:~/virtualenv# python3 -c "from pymilvus import Collection"
(milvus-binbin-1) root@milvus-client-pod:~/virtualenv#
(milvus-binbin-1) root@milvus-client-pod:~/virtualenv# pip show pymilvus
Name: pymilvus
Version: 2.1.0
Summary: Python Sdk for Milvus
Home-page: https://github.com/milvus-io/pymilvus
Author: Milvus Team
Author-email: milvus-team@zilliz.com
License: Apache-2.0
Location: /root/virtualenv/milvus-binbin-1/lib/python3.8/site-packages
Requires: grpcio, mmh3, pandas, ujson
Required-by:
```

**Comment by binbinlv** [comment]
@XuanYang-cn 

Do you know when will this error happens?

**Comment by jennyli-z** [comment]
> Actually, this issue could not be reproduced at my side:
> 
> ```
> (milvus-binbin-1) root@milvus-client-pod:~/virtualenv# python3 -m pip install pymilvus==2.1.0
> (milvus-binbin-1) root@milvus-client-pod:~/virtualenv# python3 -c "from pymilvus import Collection"
> (milvus-binbin-1) root@milvus-client-pod:~/virtualenv#
> (milvus-binbin-1) root@milvus-client-pod:~/virtualenv# pip show pymilvus
> Name: pymilvus
> Version: 2.1.0
> Summary: Python Sdk for Milvus
> Home-page: https://github.com/milvus-io/pymilvus
> Author: Milvus Team
> Author-email: milvus-team@zilliz.com
> License: Apache-2.0
> Location: /root/virtualenv/milvus-binbin-1/lib/python3.8/site-packages
> Requires: grpcio, mmh3, pandas, ujson
> Required-by:
> ```

I guess you already have some requirements installed already, I have tried with the clean python environment will get other issue. https://github.com/milvus-io/pymilvus/issues/1085

**Comment by RosieZhang12** [comment]
I came across the same issue:
<img width="570" alt="image" src="https://user-images.githubusercontent.com/106942883/182981428-9d506f21-dbce-4c33-9cbc-82f874959f4c.png">
I tried to upgrade protobuf, it said there's a compatible issue with grpcio-tools:
<img width="566" alt="image" src="https://user-images.githubusercontent.com/106942883/182981511-92319016-af36-4abd-b904-1f430e2b6f2e.png">
So I just installed grpcio-tools again and the problem was resolved:
<img width="569" alt="image" src="https://user-images.githubusercontent.com/106942883/182981609-8a1e9929-68a0-4895-a0ef-c4fc53ed2e07.png">
<img width="562" alt="image" src="https://user-images.githubusercontent.com/106942883/182981644-632da919-861e-47fa-a9f9-52e9e5d25fad.png">

**Comment by YiyunNi** [comment]
Seems like this issue can be fixed but do we need to mention the extra steps of upgrading protobuf and installing grpcio-tools in the [doc](https://milvus.io/docs/v2.1.x/install-pymilvus.md)? I think most users with a clean Python environment will encounter this issue. @XuanYang-cn

**Comment by XuanYang-cn** [comment]
available at pymilvus 2.1.1

**Comment by zeel-04** [comment]
I soved the issue by doing some simple steps.

The origin of the error.

```
        """Initialize the Milvus vector store."""
        try:
            from pymilvus import Collection, utility
        except ImportError:
            raise ValueError(
                "Could not import pymilvus python package. "
                "Please install it with `pip install pymilvus`."
            )
```

I ran from `pymilvus import Collection, utility` in a seperate file and it said `scipy` is missing. I just installed `scipy` and it got resolved.
